helloworld/src/main/java/com/hello/helloworld/HelloController.java

```diff
@@ -1,20 +1,28 @@
 package com.hello.helloworld;
 
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 public class HelloController {
-    @GetMapping("/hello")
-    public List<Student> hello() {
+    List<Student> students = List.of(
+            new Student("山田太郎", "xxx@example.co.jp", "java"),
+            new Student("橋本二郎", "xxo@example.co.jp", "java"),
+            new Student("小川三郎", "xoo@example.co.jp", "AWS"));
 
-        List<Student> students = List.of(
-                new Student("山田太郎", "xxx@example.co.jp", "java"),
-                new Student("橋本二郎", "xxo@example.co.jp", "java"),
-                new Student("小川三郎", "xoo@example.co.jp", "AWS"));
-
-        return students;
+    @GetMapping("/hello")
+    public List<Student>hello(@RequestParam(required = false) String name) {
```

Comment:
学生をあつかうのでパスやメソッド名をstudentにするのがいいかと思います。
```suggestion
    @GetMapping("/students")
    public List<Student> getStudents(@RequestParam(required = false) String name) {
```

Reply:
訂正いたしました．確認をお願いいたします．
469d0450abbb71ef9103a3c23cf7aa8188496988